xiaomi_gateway/__init__.py

```diff
@@ -50,19 +51,24 @@ def discover_gateways(self):
             host = gateway.get('host')
             port = gateway.get('port')
             sid = gateway.get('sid')
-            key = gateway.get('key')
```

Comment:
Yes, I removed it just to pass the lint check, because there are too many local variables and this variable is only used in one place. So I'll call it directly on the method argument.